src/geos_functions.jl

```diff
@@ -1031,6 +1031,58 @@ getGeomDimensions(ptr::GEOSGeom, context::GEOSContext = _context) =
 getCoordinateDimension(ptr::GEOSGeom, context::GEOSContext = _context) =
     GEOSGeom_getCoordinateDimension_r(context.ptr, ptr)
 
+# Finds the minimum X value in the geometry
+function getXMin(ptr::GEOSGeom, context::GEOSContext = _context)
+    out = Ref{Float64}()
+    result = GEOSGeom_getXMin_r(context.ptr, ptr, out)
+    if result == 0
```

Comment:
Shouldn't this be C_NULL? Same for the other methods.

Reply:
@evetion thanks for the review. `GEOSGeom_getXMin_r` returns an Int not a pointer as in other functions, see https://libgeos.org/doxygen/geos__c_8h.html#a71527bdd3c5b915f6ce428785859a466